def/sk/menu_linux.cson

```diff
@@ -0,0 +1,307 @@
+Menu:
+  "&File":
+    value: "&Súbor"
+    submenu:
+      "New &Window":
+        value: "Nové &okno"
+      "&New File":
+        value: "&Nový súbor"
+      "&Open File…":
+        value: "&Otvorť súbor…"
+      "Open Folder…":
+        value: "Otvoriť zložku…"
+      "Add Project Folder…":
+        value: "Pridať zložku projektu…"
+      "Reopen Project":
+        value: "Znovu otvoriť projekt"
+        submenu:
+          "Clear Project History":
+            value: "Vymazať históriu projektu"
+      "Reopen Last &Item":
+        value: "Znovu otvoriť poslednú &položku"
+      "&Save":
+        value: "Uložiť"
+      "Save &As…":
+        value: "Uložiť ako…"
+      "Save A&ll":
+        value: "Uložiť všetko"
+      "&Close Tab":
+        value: "Zatvoriť záložku"
+      "Close &Pane":
+        value: "Zatvoriť panel"
+      "Clos&e Window":
+        value: "Zatvoriť okno"
+      "Quit":
+        value: "Ukončiť"
+      "Close All Tabs":
+        value: "Zatvoriť všetky záložky"
+  "&Edit":
+    value: "Upraviť"
+    submenu:
+      "&Undo":
+        value: "Späť"
+      "&Redo":
+        value: "Znovu"
+      "&Cut":
+        value: "Vystrihnúť"
+      "C&opy":
+        value: "Kopírovať"
+      "Copy Pat&h":
+        value: "Kopírovať cestu"
+      "&Paste":
+        value: "Vložiť"
+      "Select &All":
+        value: "Vybrať všetko"
+      "Paste Without Reformatting":
+        value: "Vložiť bez preformátovania"
```

Comment:
Please temporarily comment L55, L56 to pass CI; This menu item currently does not support translation